### DISCUSSION TITLE
pattern filling

### DISCUSSION THREAD

**Discussion by SizNi** [cduck/drawSvg#88] [Q&A] [answered]
Hi, maybe there is a method of filling with a vector/raster drawing?

**Comment by cduck** [comment]
Not that I'm aware of but you can check [here](https://developer.mozilla.org/en-US/docs/Web/SVG) to be sure.

You could use `Image`, `Use`, and `Mask` to fake the same effect.  https://developer.mozilla.org/en-US/docs/Web/SVG will describe how they work and the examples in this repo show how to use them.

**Reply by SizNi** [reply]
Thank you!
 I will try

**Comment by cduck** [comment, marked as answer]
Correction.  You can use the SVG `<pattern>` tag with an `Image` inside.  drawSvg currently does not implement this tag so you can either implement it yourself or install the pre-release of drawsvg 2.0 (#89).

**Reply by SizNi** [reply]
Thank you!

**Reply by cduck** [reply]
Update: 2.0 is released.  See [this example](https://github.com/cduck/drawSvg#patterns-and-gradients).

**Reply by SizNi** [reply]
Thank you! i solve in vector way, examples here: https://github.com/SizNi/geol_project
